api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/BaseHandler.java

```diff
@@ -51,6 +56,17 @@ public Response baseJSON(@Context ServletContext context) {
         return Response.ok(hawkularMetrics).build();
     }
 
+    @GET
+    @Produces({APPLICATION_XHTML_XML, TEXT_HTML})
+    @ApiOperation(value = "Returns some basic information about the Hawkular Metrics service.",
+                  response = String.class, responseContainer = "Map")
+    public void baseHTML(@Context ServletContext context) throws Exception {
+
+        HttpServletRequest request = ResteasyProviderFactory.getContextData(HttpServletRequest.class);
```

Comment:
You can't use @Context here, @Context has to returns a proxy object and the RequestDispather.forward(request, response) will only accept the original objects. See https://issues.jboss.org/browse/RESTEASY-745

I could put index.html in the root, but the images and css would either have to stay into the /static directory or we would have to add more logic to the BaseHandler for each of those files. I believe it makes more sense to keep all of the static content under /static and use the servlet mapping to handle all the static files.